go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/kubeslice/apis v0.1.3
 	github.com/kubeslice/gateway-sidecar v0.2.0
-	github.com/kubeslice/kubeslice-monitoring v0.1.6
+	github.com/kubeslice/kubeslice-monitoring v0.1.7-0.20230404135238-9338c8ec16f9
```

Comment:
nit: can we release a tag from monitoring repo and use it here ?

Reply:
yes. i will do it in the next pr